provider.go

```diff
@@ -36,20 +37,39 @@ func (p *Provider) GetRecords(ctx context.Context, zone string) ([]libdns.Record
 		return nil, err
 	}
 
-	reqURL := fmt.Sprintf("%s/zones/%s/dns_records", baseURL, zoneInfo.ID)
-	req, err := http.NewRequestWithContext(ctx, "GET", reqURL, nil)
-	if err != nil {
-		return nil, err
-	}
+	page := 1
+	perPage := 100 // Maximum allowed by Cloudflare API
```

Comment:
```suggestion
	const perPage = 100 // Maximum allowed by Cloudflare API
```

Or maybe `pageSize` or even `maxPageSize`?

Reply:
sounds good, I set it to `maxPageSize`.